cmd/collector/app/builder_flags.go

```diff
@@ -102,8 +103,15 @@ func (cOpts *CollectorOptions) InitFromViper(v *viper.Viper) *CollectorOptions {
 	cOpts.DynQueueSizeMemory = v.GetUint(collectorDynQueueSizeMemory) * 1024 * 1024 // we receive in MiB and store in bytes
 	cOpts.NumWorkers = v.GetInt(collectorNumWorkers)
 	cOpts.QueueSize = v.GetInt(collectorQueueSize)
-	cOpts.TLSGRPC = tlsGRPCFlagsConfig.InitFromViper(v)
-	cOpts.TLSHTTP = tlsHTTPFlagsConfig.InitFromViper(v)
-
-	return cOpts
+	TLSGRPC, err := tlsGRPCFlagsConfig.InitFromViper(v)
+	if err != nil {
+		return cOpts, errors.New("ES_TLS_ENABLED is set to false unable to talk to server(s)")
```

Comment:
same issue - the message is "Failed to initialize TLS", we don't know why at this point. And you need to capture the original error (https://blog.golang.org/go1.13-errors):
```
fmt.Errorf("Failed to initialize TLS: %w", err)
```